src/main/jbake/content/community/docs/developer-guide/alerts-v2.adoc

```diff
@@ -872,6 +872,253 @@ Url can be a list of valid link:https://hc.apache.org/httpcomponents-core-ga/htt
 | Used for X-Proxy-Remote-User HTTP header.
 |=======================
 
+== Prometheus Integration
+
+=== Prometheus Alerter
+
+The Prometheus Alerter will listen for triggers tagged with "prometheus" tag. the Prometheus evaluations and interacts with the Alerts system. +
```

Comment:
I'm not sure what you are trying to say here, it looks maybe like something that you mean to remove?

> the Prometheus evaluations and interacts with the Alerts system.

Reply:
I accidentaly chopped the text.
The description is just a moving from the class javadoc into here.